### ISSUE TITLE
False negative: no error reported on fully qualified map using unknown alias

### ISSUE THREAD

**Issue by keychera** [clj-kondo/clj-kondo#2676]
- [x] I have read the [Clojure etiquette](https://clojure.org/community/etiquette) and will respect it when communicating on this platform.

**version**

clj-kondo v2025.09.22

**macro usage**

no macro

**platform**

native-version and clj-kondo bundled with Calva

**editor**

Calva

**problem**

 fully qualified map using alias that has missing :require doesn't compile but this doesn't produce any lint error

**repro**

```clojure
(ns test
  #_(:require
     [gl.shader :as shader]
     [model.assimp :as assimp]))

[::wirecube
 #::assimp{:model-to-load ["assets/cube.glb"] :tex-unit-offset 0}
 #::shader{:use ::barycentric-shader}]
```
lint result
```
clj-kondo.exe --lint .\test.clj 
linting took 240ms, errors: 0, warnings: 0
```

**config**

no special config

**expected behavior**

error or warning reported the same way as using fully qualified keys
```
[::wirecube
 {::assimp/model-to-load ["assets/cube.glb"] :tex-unit-offset 0}
 {::shader/use ::barycentric-shader}]
```
lint result
```
.\test.clj:2:3: warning: Unresolved namespace assimp. Are you missing a require?
.\test.clj:3:3: warning: Unresolved namespace shader. Are you missing a require?
linting took 207ms, errors: 0, warnings: 2
```
